packages/login/src/components/PasswordRequest/PasswordRequest.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import {FormattedMessage, intlShape} from 'react-intl'
+import {Pages} from '../../types/Pages'
+import * as Tocco from 'tocco-ui'
+
+import '../Login/styles.scss'
+
+export class PasswordRequest extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      username: ''
+    }
+  }
+
+  handleSubmit(e) {
+    e.preventDefault()
+    this.props.requestPassword(this.state.username)
+  }
+
+  handleUsernameChange(e) {
+    this.setState({
+      username: e.target.value
+    })
+  }
+
+  render() {
+    return (
+      <div className="login-form">
+        {this.props.showTitle && <h1><FormattedMessage id="client.login.passwordRequest.title"/></h1>}
+        <p><FormattedMessage id="client.login.passwordRequest.introduction"/></p>
+        <form>
```

Comment:
I just tested this form (in our Nice environment). If the abort button is clicked, the form gets submitted. This is certainly not desired. Should be fixed as follows:

- handleSubmit should be registered on the form
- the submit button should be of type `submit` (without click handler)
- the abort button should be of type `button`